service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -450,6 +530,21 @@ public void removeWorkspaceRole(
     }
   }
 
+  /** Wrapper around Sam client to fetch the list of users with a specific role in a workspace. */
+  @Traced
+  public List<String> listWorkspacePolicyMembers(
```

Comment:
I'd name this `listUsersWithWorkspaceRole` or similar, since policies are a Sam concept.

Reply:
Good point